app/components/Header/DropdownContents/EventsDropdown/EventsDropdown.css

```diff
@@ -0,0 +1,22 @@
+.eventsDropdownEl {
+  width: 20rem;
+}
+
+.dropdownLink {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
```

Comment:
This looks really wonky to me IMO. Don't know how others feel, but the fact that the icons and text are misaligned is really weird. Maybe use grid for this one? That way we can have the icons and link text column-aligned.

Reply:
I totally agree, and the plan is to have everything nicely aligned! I've just been struggling with the design, and needed a small break to gain some creativity 😄